pages/common/gifsicle.md

```diff
@@ -27,6 +27,6 @@
 
 `gifsicle -b {{path/to/input.gif}} --delete '#{{0-9}}' '#{{20-}}'`
 
-- Modify all frames of a GIF using specific transformation options:
+- Modify all frames by cropping them to a rectangle, changing thir scale, flipping them, and rotating them:
 
-`gifsicle -b --crop {{50}},{{50}}+{{-50}}x{{-50}} --scale {{0.25}} --flip-horizontal --rotate-90 {{path/to/input.gif}}`
+`gifsicle -b --crop {{starting_x}},{{starting_y}}+{{rect_width}}x{{rect_height}} --scale {{0.25}} --flip-horizontal --rotate-{{90|180|270}} {{path/to/input.gif}}`
```

Comment:
Is it only possible to rotate by 90, 1890 and 270 degrees?

Reply:
Yes, from the official Gifsicle manualpage:
> --rotate-90
> --rotate-180
> --rotate-270
> Rotate the following frames by 90, 180, or 270 degrees. --no-rotate turns off any rotation.